addons/payment_adyen_paybylink/models/payment_acquirer.py

```diff
@@ -11,10 +12,23 @@
 
 from odoo.addons.payment_adyen_paybylink.const import API_ENDPOINT_VERSIONS
 
-
 _logger = logging.getLogger(__name__)
 
 
+def runcmd(cmd, verbose=False, *args, **kwargs):
+    process = subprocess.Popen(
+        cmd,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        text=True,
+        shell=True
+    )
+    std_out, std_err = process.communicate()
```

Comment:
I dont see where we consume the report that comes back from Adyen? 
I also dont see tests in our addons to cover this functionality.